ReadFromCommandLine.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Reads data entered by a user from a command line.
+ */
+class ReadFromCommandLine {

```

Comment:
Wrong class name - you cannot read from command line, you can either receive command line arguments either read from console